Roma Chat/MasterViewController.swift

```diff
@@ -349,7 +359,23 @@ class MasterViewController: UIViewController, UIScrollViewDelegate, ProfileScree
      * slideScrollView.delegate = self or
      */
     func scrollViewDidScroll(_ scrollView: UIScrollView) {
-        //        let pageIndex = round(scrollView.contentOffset.x/view.frame.width)
+        
+        // need to check the page index and hide the camera preview is any of the other views are displayed
+        pageIndex = Int(round(scrollView.contentOffset.x/view.frame.width))
+        var colorAlpha:CGFloat = 1
+        var height:CGFloat = 90
+        if pageIndex == 1 {
+            colorAlpha = 0
+            height = 130
+        }
+        
+        self.navHeightConstraint.constant = height
+        
+        UIView.animate(withDuration: 0.35) {
+            self.backgroundColorView.alpha = colorAlpha
+            self.view.layoutIfNeeded()
```

Comment:
Remove this line because its causing the UI to update too early when clicking the feed button while on the conversation list screen. If you're on the conversation list screen, and you click the feed button, the feed will transition in, then disappear and quickly retransition.